frontend/src/tabs/zoom.pug

```diff
@@ -1,8 +1,14 @@
 #zoom
+  span.large-font Commits Panel
   .zoom__title
     .zoom_title--group.large-font
-      span(v-if="info.filterGroupSelection === 'groupByAuthors'") {{ info.user.displayName }} ({{ info.user.name }})
-      span(v-else) {{ info.user.repoName }}
+       span(v-if="info.filterGroupSelection === 'groupByAuthors'") {{ info.user.displayName }} ({{ info.user.name }})
```

Comment:
It seems that you forgot my comment on the indentation. We use 2 space indentation for `pug` files.

Reply:
Ah sorry about that. Seemed to have missed it